packages/js/api-client/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@openverse/api-client",
+  "version": "0.0.1-a2",
+  "description": "API client for the Openverse REST API",
+  "main": "dist/index.cjs",
+  "module": "dist/index.mjs",
+  "types": "dist/index.d.ts",
+  "sideEffects": false,
+  "type": "module",
+  "exports": {
+    "import": "./dist/index.mjs",
+    "require": "./dist/index.cjs"
+  },
+  "scripts": {
+    "build": "rm -rf dist && mkdir dist && pnpm run build:cjs && pnpm run build:esm && pnpm run build:types && pnpm pack && pnpm publish --dry-run --no-git-checks",
+    "build:cjs": "esbuild ./src/**/* --packages=external --sourcemap --format=cjs --out-extension:.js=.cjs --outdir=dist",
+    "build:esm": "esbuild ./src/**/* --packages=external --sourcemap --format=esm --out-extension:.js=.mjs --outdir=dist",
+    "build:types": "tsc -p ./tsconfig.prod.json",
+    "types": "tsc -p .",
+    "test:unit": "vitest run",
+    "generate": "node ./bin/generate.mjs"
+  },
+  "keywords": [
+    "openverse",
+    "creative commons"
+  ],
+  "homepage": "https://docs.openverse.org/packages/js/api_client/index.html",
+  "bugs": {
+    "url": "https://github.com/WordPress/openverse/issues"
+  },
+  "author": "Openverse Contributors <openverse@wordpress.org>",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/WordPress/openverse",
+    "directory": "packages/js/openverse-api-client"
+  },
+  "license": "MIT",
+  "devDependencies": {
+    "esbuild": "^0.23.1",
+    "nock": "14.0.0-beta.11",
```

Comment:
Nock only supports fetch as of version 14, which is still in beta. It works fine for these test cases, so I don't think it will be an issue to be pinned to a pre-release version for now.